.travis.yml

```diff
@@ -1,18 +1,15 @@
 language: php
 
 php:
-    - 5.3
-    - 5.4
-    - 5.5
-    - 5.6
-    - hhvm
+    - 7.2
```

Comment:
The `php-7.1` is stable. 

I think we should add the `php-7.1` version in Travis CI build.